eox_tenant/signals.py

```diff
@@ -0,0 +1,187 @@
+# -*- coding: utf-8 -*-
+"""
+Eox Tenant Signals
+==================
+This module makes it possible to override the settings object for a tenant.
+
+How to debug in devstack
+========================
+
+Add the following lines to the lms.envs.devstack_docker module
+
+LOGGING['handlers']['console']['level'] = 'DEBUG'
+LOGGING['loggers']['']['level'] = 'INFO'
+LOGGING['loggers']['eox_tenant'] = {
+    'handlers': ['console'],
+    'propagate': False,
+    'level': 'DEBUG'
+}
+
+"""
+from __future__ import print_function, unicode_literals
+
+import logging
+import os
+import six
+
+from datetime import datetime
```

Comment:
@felipemontoya I think that this could be placed on standard lib imports.

Reply:
Yes, this was marked by pylint as well